test/all_test.html

```diff
@@ -11,6 +11,8 @@
     <title>Dart CodeMirror Tests</title>
     <link rel="x-dart-test" href="all_test.dart">
     <script src="packages/test/dart.js"></script>
+    <!-- Required dependency for merge addon -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/diff_match_patch/20121119/diff_match_patch.js"></script>
```

Comment:
No. We include the full release in the bundle, we don't rely on CloudFlare's CDN. See [updating_codemirror.md](https://github.com/google/codemirror.dart/blob/master/updating_codemirror.md) for the update process.

Reply:
Thanks for the PR!

wrt what @domesticmouse - should this script reference be moved down to ~line 20 (`<script src="packages/codemirror/codemirror.js...`), and be converted to a relative reference?

Reply:
Thanks, I've read the updating_codemirror.md file, but it did not say anything regarding extra dependencies that are from a different JS package than codemirror is.

I was thinking in download the most recent JS [diff-match-patch](https://github.com/google/diff-match-patch) and put it on the `third_party` folder. Does that make sense?

Reply:
I'm not up to speed on the legals of including Apache libs into a BSD 3 codebase, but this sounds like a good plan to me.